with open("a_file.txt") as file:
    file.read()

#  File "/Users/carlosdelgado/Desktop/HunidDaysOfPython/Intermediate/Day30/errors/main.py", line 1, in <module>
#     with open("a_file.txt") as file:
#  FileNotFoundError: [Errno 2] No such file or directory: 'a_file.txt'
